### PR TITLE
Add print of the priors in tutorial 

### PR DESCRIPTION
**Pull request by AtreyeeS**
This adds an example of printing a parametrs table with priors in our tutorials.
In general, it might be useful to add some analysis examples with priors

### CODE REVIEW

**Review by bkhelifi — DISMISSED** [review]
;) thanks Atreyee!

File: examples/tutorials/api/priors.py [hunk 1]
```diff
@@ -192,6 +192,14 @@
...
 
 
+""
+# The parameters table will mention the type of prior associated to each model
+print(results_prior.models.to_parameters_table())
+
+""
+# To see the details of the priors, eg:
+print(results_prior.models.parameters["index"].prior)
+
```
Comment (Astro-Kirsty) [inline examples/tutorials/api/priors.py:202]:
```suggestion
""
######################################################################
# The parameters table will mention the type of prior associated to each model

print(results_prior.models.to_parameters_table())


######################################################################
# To see the details of the priors, eg:

print(results_prior.models.parameters["index"].prior)

```

Comment (Astro-Kirsty) [reply examples/tutorials/api/priors.py:202]: image

**Review by Astro-Kirsty — APPROVED** [review]

**Review by registerrier — APPROVED** [review]
Thanks @AtreyeeS . Looks good!

**Pull request merged**